.github/workflows/playwright.yml

```diff
@@ -0,0 +1,30 @@
+name: Playwright Tests
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  test:
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+      - name: Install dependencies
+        run: yarn
+      - name: Install Playwright Browsers
+        run: npx playwright install --with-deps
```

Comment:
Shouldn't we add this as a command we can run to setup playwright locally? Otherwise, we need to encounter the error, check the log & make sure it's normal to have this error + run the command manually.

Reply:
To add to this, on the DS we lock the version we install, if the repo used `v1.2.1` but `v2.0.0` was out, would this command install that version causing potential changes? 🤔

Reply:
Hum 🤔  I think they are auto install in the postinstall script. The workflow was auto generated by playwright I didn't look at it :D

Reply:
@joshuaellis 🤔  this command install the playwright browsers not playwright itself. I'm not sure I understand ? 

@Convly You might be right I'm not seeing much documented on this I can probably add a shortcut for this but mostly I'll add this in the contributing guide once we have a good few feedback and I make the changes 👍